src/less/less/ds6/variables.less

```diff
@@ -270,3 +270,12 @@
 //------------------------------------
 
 @ds6-outline-dotted: 1px dotted @ds6-color-g205-gray;
+
+// Fixed Height Values for Textbox, Combobox & Button
+// (size are PX intentionally, and should not scale with root font size)
+//-----------------------------
+
+@height-control-small: 40px;
```

Comment:
I believe this needs to be 32px;

Also, these should be prefixed with `@ds6-`.

Reply:
There shouldn't be any small size anymore. According to the design team this is now the same as medium.  But for backwards compatibility I didn't want to get rid of it so I just changed it to match the medium size.

This will also let us easily add it back when the design team decides they need it after all.

I didn't prefix it with `@ds6` because they are used in base less files too.  Is that OK?

Reply:
What if we did `@ds6-form-control-large`?

Reply:
Or, `@ds6-form-control-height-large`, etc.?

Reply:
I'm using the existing names, but I could rename them to `form-control-height-x` to make it clearer that they are just used for form controls.

But no matter the name, the variable is used in the base button less file, so I can't prefix it with `ds6`.  Unless you want me to move the usage out of the base button less file and into the DS specific files.

Reply:
Ah, I see what you're saying now. That's how the other components work, I believe: the height settings are not in the base files, but rather in the individual DS-specific files.

Reply:
Button must be the outlier at the moment.

Reply:
Do you want me to update all these variables to have DS-specific prefixes, and then move the button reference out of the base file?

Reply:
FYI, there are other "non DS-specific" variable names in this file already... like:
@font-weight-thin
@color-interface-inline-text-edit

So I think its OK as is